code-challenges.js

```diff
@@ -66,68 +35,26 @@ console.log(multiplyer);
+
+
 
 
 // --------------------5) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
 // Copy and paste or rewrite the function. Use the test variables provided below.
 
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
+var notAString1 = true
+var notAString2 = 42
+
+
 
-//create const variable connected to the function and put the term string within the parenthesis
-const noVowels = (string) => {
-  //create a let variable stating what vowels are
- let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
- //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
- let array = string.split("")
- //create a return statement that will filter through the array and add in .join to join the constanants together
- return array.filter(value => vowels.indexOf(value) === -1).join("")
-//}
-//at this point I think I need to change it to a type of type statement? so that way it would go thorugh and give me a boolean OR number... but IDK how to do that....
-console.log() both variables given
-console.log(noVowels(stringWithVowels1));
-console.log(noVowels(stringWithVowels2))
 
 // --------------------6) Create a function that takes the toonimals array and returns an array with only the toon objects that are cats.
 // Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
 
-//make all variables stackes above one another
-var toonimals = [
-  {name: "Itchy", animal: "mouse"},
-  {name: "Stimpy", animal: "cat"},
-  {name: "Daffy", animal: "duck"},
-  {name: "Scratchy", animal: "cat"},
-  {name: "Ren", animal: "dog"},
-  {name: "Felix", animal: "cat"}]
-//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find "cats"
-var animal = toonimals.filter(value => value.animal === "cat")
-//console.log() the code to call it
-console.log(animal);
+
+var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
+
+
 
 // --------------------7) Using the toonimals variable from #6, create a function that returns only the names of the non-cats.
 // Expected output: "Itchy" "Daffy" "Ren"
-//make all variables stackes above one another
-var toonimals = [
-  {name: "Itchy", animal: "mouse"},
-  {name: "Stimpy", animal: "cat"},
-  {name: "Daffy", animal: "duck"},
-  {name: "Scratchy", animal: "cat"},
-  {name: "Ren", animal: "dog"},
-  {name: "Felix", animal: "cat"}]
-//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find non cat names
-var animal = toonimals.filter(value => value.animal !== "cat")
-//console.log() the code to call it
-console.log(animal);
```

Comment:
almost! not quite but you're almost there. this solution again, is just a variable, not a function. also, it will return the whole object of each non-cat. we want just the _names_ of the non-cats